eng/Packages.props

```diff
@@ -28,6 +28,7 @@
     <PackageReference Update="System.IO.Compression" Version="4.3.0" />
     <PackageReference Update="System.IO.FileSystem.Primitives" Version="4.3.0" />
     <PackageReference Update="System.Linq.Parallel" Version="4.0.1" />
+    <PackageReference Update="System.Memory" Version="4.5.2" />
```

Comment:
I was just curious about the version here. Looks good.

https://devdiv.visualstudio.com/DevDiv/_git/VS?path=%2Fsrc%2Fappid%2Fdevenv%2Fstub%2Fdevenv.urt.config.tt&version=GBmaster&line=1505&lineStyle=plain&lineEnd=1509&lineStartColumn=9&lineEndColumn=29

That's `4.0.1.0` which is the assembly version of this package. You may have already gone through this though :)